samples/get_view_image.py

```diff
@@ -0,0 +1,67 @@
+####
+# This script demonstrates how to use the Tableau Server Client
+# to query a high resolution image of a view from Tableau Server.
+#
+# For more information, refer to the documentations on 'Query View Image'
+# (https://onlinehelp.tableau.com/current/api/rest_api/en-us/help.htm)
+#
+# To run the script, you must have installed Python 2.7.X or 3.3 and later.
+####
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Query View Image From Server')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--view-name', '-v', required=True, help='name of view')
```

Comment:
can we expand the help text just a bit please? something like "name of the view we want an image of"